src/home/WalletHome.tsx

```diff
@@ -56,6 +58,16 @@ function WalletHome() {
   const showNotificationCenter = getFeatureGate(StatsigFeatureGates.SHOW_NOTIFICATION_CENTER)
   const showNotificationSpotlight = showNotificationCenter && canShowNotificationSpotlight
 
+  const [hideBalances, setHideBalances] = useState(false)
```

Comment:
Also, I just realized this. Should this state be persisted across sessions? E.g., if someone wants to have their balances hidden when Valora is launched

Reply:
I'm not sure it makes sense to persist it. In that case a user would have to decide before they leave the app that next time they open it they want it to be hidden, which I don't know how likely that is. We could have the initial state be true so that it is hidden by default?

Reply:
+1 persisting this setting in redux. This could be handled similar to `hapticFeedbackEnabled` in the app state.